packages/camera/camera/example/ios/RunnerTests/CameraTestUtils.h

```diff
@@ -0,0 +1,18 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+@import camera;
+
+NS_ASSUME_NONNULL_BEGIN
+
+/// Creates an `FLTCam` that runs its capture session operations on a given queue.
+/// @param captureSessionQueue the capture session queue
+/// @return an FLTCam object.
+extern FLTCam *FLTCreateFLTCamWithCaptureSessionQueue(dispatch_queue_t captureSessionQueue);
```

Comment:
Optional nit: I don't think repeating FLT the second time is all that useful here; `FLTCreateCamWith...` seems clear in context.

(Modulo `Cam` not being a great name for a class in the first place.)

Reply:
Will remove `FLT`. 

I don't like the `Cam` name either, but will keep it for now in this PR.